cumulusci/core/tests/test_config.py

```diff
@@ -1702,6 +1702,100 @@ def test_is_person_accounts_enabled__is_enabled(self):
             config._is_person_accounts_enabled, config.is_person_accounts_enabled
         )
 
+    @responses.activate
+    def test_is_multi_currency_enabled__not_enabled(self):
+        config = OrgConfig(
+            {
+                "instance_url": "https://example.com",
+                "access_token": "TOKEN",
+                "id": "OODxxxxxxxxxxxx/user",
+            },
+            "test",
+        )
+        self.assertIsNone(
+            config._is_multi_currency_enabled,
+            "_is_multi_currency_enabled should be initialized as None",
+        )
```

Comment:
Can we use pytest-style assertions please? The codebase is inconsistent, but we are trying to be consistent about using them for new tests.

    assert config._is_multi_currency_enabled is None